src/openrct2/platform/posix.c

```diff
@@ -282,10 +282,22 @@ bool platform_lock_single_instance()
         log_warning("Cannot open lock file for writing.");
         return false;
     }
-    if (flock(pidFile, LOCK_EX | LOCK_NB) == -1) {
+    #define   LOCK_SH   1    /* shared lock */
+    #define   LOCK_EX   2    /* exclusive lock */
+    #define   LOCK_NB   4    /* don't block when locking */
+    #define   LOCK_UN   8    /* unlock */
+    
+    struct flock lock;
```

Comment:
?

Reply:
In C, `struct` is required before definitions unless typedefed.